components/ws-daemon/pkg/daemon/daemon.go

```diff
@@ -169,11 +170,17 @@ func NewDaemon(config Config) (*Daemon, error) {
 
 	mgr, err = ctrl.NewManager(restCfg, ctrl.Options{
 		Scheme:                 scheme,
-		Port:                   9443,
-		Namespace:              config.Runtime.KubernetesNamespace,
 		HealthProbeBindAddress: "0",
-		MetricsBindAddress:     "0", // Metrics are exposed through baseserver.
-		NewCache:               cache.MultiNamespacedCacheBuilder([]string{config.Runtime.KubernetesNamespace, config.Runtime.SecretsNamespace}),
+		Metrics:                metricsserver.Options{BindAddress: ":0"}, // Metrics are exposed through baseserver.
```

Comment:
```suggestion
		Metrics:                metricsserver.Options{BindAddress: "0"}, // Metrics are exposed through baseserver.
```
otherwise the metrics server isn't disabled